aws-storage-s3/src/androidTest/java/com/amplifyframework/storage/s3/AWSS3StorageUploadAccessLevelTest.java

```diff
@@ -88,9 +88,8 @@ public void setUp() throws Exception {
         mobileClient.signOut();
 
         // Create a new file to upload
-        File fileToUpload = new RandomTempFile(UPLOAD_SIZE);
-        fileName = fileToUpload.getName();
-        filePath = fileToUpload.getPath();
+        uploadFile = new RandomTempFile(UPLOAD_SIZE);
+        fileName = uploadFile.getName();
```

Comment:
This existed before this PR -- but just noting it since I saw it:
`fileName` is a weird name for the first argument, in these various calls to `SynchronousStorage#uploadFile(String, File, StorageOptions)`. I think it should be named `remoteKey`, right?

Reply:
Cool. I thought perhaps there was a reason for this difference I was missing. Renamed.